Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -1463,7 +1511,84 @@ def is_ip_valid(s, accept_v6_ips=False):
         return True
 
 
-def return_outputs(readable_output, outputs=None, raw_response=None):
+class DBotScoreEntry:
```

Comment:
this is not an entry, main use is the context feature

Reply:
I had naming collusion.
DBotScore class for the context or DBotScore for enum.
Or I can combine them together.
I would like to know your thoughts

Reply:
Not sure we need this class, you only use the to_context section of it